assets/js/googlesitekit-main-dashboard.js

```diff
@@ -37,7 +37,7 @@ import DashboardEntryPoint from './components/DashboardEntryPoint';
 // Initialize the app once the DOM is ready.
 domReady( () => {
 	if ( global._googlesitekitLegacyData.admin.resetSession ) {
-		clearWebStorage();
+		clearCache();
```

Comment:
Following my previous comment, this is an example where the function isn't async but thinking again we should probably use this consistently and await it anyways since we're following this call with others. The only time this shouldn't need to be awaited would be as the only function attached to an event handler for example.

Reply:
Understood. Addressed, thank you for pointing it out!